apps/web/src/components/search/search.css

```diff
@@ -1,3 +1,8 @@
 #searchInput:focus + #searchIcon {
   color: var(--primary);
 }
+.darkModeSearch:-webkit-autofill{
+  -webkit-text-fill-color: rgba(231, 225, 225, 0.842) !important;
```

Comment:
What is this color?

Reply:
It matches the original search box color

Reply:
rgba(231, 225, 225, 0.842)

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-fill-color this isn't needed. The browser will automatically fall back to using the `color` property.

Reply:
Without using the webkitFillColor the text colors are not changing..